src/promptflow/promptflow/executor/flow_validator.py

```diff
@@ -73,23 +77,28 @@ def _validate_nodes_topology(flow: Flow) -> Flow:
         for node in flow.nodes:
             if node.name in node_names:
                 raise DuplicateNodeName(
-                    message=f"Node with name '{node.name}' appears more than once in the node definitions in your "
-                    f"flow, which is not allowed. To address this issue, please review your "
-                    f"flow and either rename or remove nodes with identical names.",
+                    message_format=(
+                        "Flow is defined incorrectly. Node with name '{node_name}' appears more than once "
```

Comment:
Do we really need this sentence "Flow is defined incorrectly"? I guess description + proposal is enough?